### PR TITLE
Bug 1119164 - [TextSelection] separate marionette test pages by suite

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=1119164

### CODE REVIEW

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4c9066d3e761)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=527bce1e78c2)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=362d44b7d947)

File: apps/system/js/text_selection_dialog.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* global layoutManager, SettingsListener */
 'use strict';
 (function(exports) {
-  var DEBUG = false;
+  var DEBUG = true;
```
Comment (alivedise) [inline apps/system/js/text_selection_dialog.js]: false

File: apps/system/test/marionette/lib/faketextselectionapp.js [hunk 2]
```diff
@@ -14,13 +14,21 @@ function FakeTextSelectionApp(client) {
...
 FakeTextSelectionApp.Selector = Object.freeze({
-  bottomLeftInput: '#bottom-left-input',
-  bottomRightInput: '#bottom-right-input',
-  centerInput: '#center-input',
-  topLeftInput: '#top-left-input',
-  topRightInput: '#top-right-input',
-  normalDiv: '#noneditable',
-  nonSelectedDiv: '#noneditable-userselectnone'
+  // dialogposition.html
+  DPbottomLeftInput: '#bottom-left-input',
```
Comment (alivedise) [inline apps/system/test/marionette/lib/faketextselectionapp.js]: nit: I would suggest not abbr. it; DialogPosition/Functionality/Noneditable/Bug(?)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=537a557410e9)

**Pull request merged**